pom.xml

```diff
@@ -22,10 +22,24 @@
     </licenses>
 
     <dependencies>
+      <!--
+        TODO: ??
+        After updating parent pom version dep to 1.580.1, we needed to explicitly include
+        the remoting jar before maven-plugin. If not, we get a 'java.lang.SecurityException: class
+        "org.jenkinsci.remoting.CallableDecorator"'s signer information does not match signer
+        information of other classes in the same package' exception. Looks to be related to the
+        addition of org.jenkins-ci:SECURITY-144-compat:jar:1.0, which the maven plugin depends on
+        and has classes in the org.jenkinsci.remoting package.
+      -->
```

Comment:
This is [JENKINS-25625](https://issues.jenkins-ci.org/browse/JENKINS-25625) (CC @kohsuke).